installer/yaml/resources/strimzi-operator/strimzi-operator.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  ownerReferences:
+    - apiVersion: cloudflow-installer.lightbend.com/v1alpha1
+      blockOwnerDeletion: true
+      controller: true
+      kind: Cloudflow
+      name: __instanceId__
+      uid: __instanceUid__
+  name: strimzi-cluster-operator
+  labels:
+    app: strimzi
+    chart: strimzi-kafka-operator-0.16.2
+    component: deployment
+    release: cloudflow
+    heritage: Helm
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: strimzi-cluster-operator
+      strimzi.io/kind: cluster-operator
+  template:
+    metadata:
+      labels:
+        name: strimzi-cluster-operator
+        strimzi.io/kind: cluster-operator
+    spec:
+      serviceAccountName: strimzi-cluster-operator
+      containers:
+        - name: strimzi-cluster-operator
+          image: __strimziOperator.imageName__:__strimziOperator.imageTag__
+          args:
+            - /opt/strimzi/bin/cluster_operator_run.sh
+          env:
+            - name: STRIMZI_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: STRIMZI_FULL_RECONCILIATION_INTERVAL_MS
+              value: "120000"
+            - name: STRIMZI_OPERATION_TIMEOUT_MS
+              value: "300000"
+            - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+              value: __kafka.imageName__:__kafka.imageTag__
+            - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
+              value: __kafka.imageName__:__kafka.imageTag__
+            - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
+              value: __kafka.imageName__:__kafka.imageTag__
+            - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
+              value: __kafka.imageName__:__kafka.imageTag__
+            - name: STRIMZI_KAFKA_IMAGES
+              value: |
+                2.3.1=strimzi/kafka:0.16.2-kafka-2.3.1
```

Comment:
Is it possible to add some comment here that we choos 2.4.0 and don't worry about 2.3.1?

Reply:
Comment added.